pages/api/restricted.js

```diff
@@ -0,0 +1,17 @@
+import { getServerSession } from "next-auth/next";
+import { authOptions } from "./auth/[...nextauth]";
+
+export default async function handler(req, res) {
```

Comment:
It's a good practice not to use abbreviations as variable names.

And where is this endpoint used? Was it just for test purposes?

Reply:
was for test purposes only